doc/dev/BUILD.md

```diff
@@ -45,3 +45,21 @@ Some modules, plugins and bindings depending on external libraries can be option
 * `F3D_PLUGIN_BUILD_DRACO`: Support for DRC file format. Requires `Draco`. Disabled by default.
 * `F3D_BINDINGS_PYTHON`: Generate python bindings (requires `Python` and `pybind11`). Disabled by default.
 * `F3D_BINDINGS_JAVA`: Generate java bindings (requires `Java` and `JNI`). Disabled by default.
+
+## Installing
+
+Installation can be done through CMake, by running the following command:
+
+```
+cmake --build ${your_build_dir}
+```
+
+Optional components can be installed by specifying the component name:
+
+```
+cmake --build ${your_build_dir} --component ${component_name}
+```
+
+Here is the list of the optional components you can install:
```

Comment:
mention `all_components` somewhere ?

Reply:
Do you want to list of all the components installed by default?

Reply:
No, I mean that mention that you can install all components easily by using all_components

Reply:
There is a misunderstanding. There is no `all_components` group. If `--component` is not specified, all the components not flagged as `EXCLUDE_FROM_ALL` are installed.

Reply:
So, once this is merged, someone compiling F3D wanting to install all components of F3D has no way to know the list of component to potentially install apart reading the CMakeLists.txt ?

Reply:
Currently, if the user specify nothing (ie `cmake --install .`), everything is installed, except `mimetypes` component.  
If he wants to install it, then yes, the user has to know the name of the component (if there is a way to get the components name, I'm not aware of it).

Reply:
Then all of these optional component should be at least documented and packagers will need to be informed. I'm afraid that some package will miss the mimetypes in the next release.

Reply:
Not a big fan but I see the point of it.

Reply:
I find it much cleaner than a CMake configuration-time option (which is not possible in case anyway)  
I'd rather exclude all optional components from `ALL`, and let the packager install what he wants.

Reply:
Yes I agree it is better, I'd just hope to have a way to make sure nobody miss out on a feature because of it.